src/store/modules/wallet/wallet.js

```diff
@@ -57,6 +57,9 @@ export default {
 		setBackupState(state, backupState) {
 			state.backupState = backupState
 		},
+		addMessageType(state, {typeUrl,type}) {
+			state.activeClient.registry.register(typeUrl,type) 
```

Comment:
Directly mutating a private property of the signing client as a workaround for now. Subject to change, see: https://github.com/cosmos/cosmjs/issues/628